packages/vm/test/api/EIPs/eip-7685.spec.ts

```diff
@@ -0,0 +1,99 @@
+import { Block } from '@ethereumjs/block'
+import { Blockchain } from '@ethereumjs/blockchain'
+import { Chain, Common, Hardfork } from '@ethereumjs/common'
+import { CLRequest, KECCAK256_RLP, concatBytes, randomBytes } from '@ethereumjs/util'
+import { assert, describe, expect, it } from 'vitest'
+
+import { VM } from '../../../src/vm.js'
+import { setupVM } from '../utils.js'
+
+import type { CLRequestType } from '@ethereumjs/util'
+
+class NumberRequest extends CLRequest implements CLRequestType {
+  constructor(type: number, bytes: Uint8Array) {
+    super(type, bytes)
+  }
+
+  public static fromRequestData(bytes: Uint8Array): CLRequestType {
+    return new NumberRequest(0x1, bytes)
+  }
+
+  serialize() {
+    return concatBytes(Uint8Array.from([this.type]), this.bytes)
+  }
+}
+
+const common = new Common({ chain: Chain.Mainnet, hardfork: Hardfork.Cancun, eips: [7685] })
+
+describe('EIP-7685 runBlock tests', () => {
+  it('should not error when a valid requestsRoot is provided', async () => {
+    const vm = await setupVM({ common })
+    const emptyBlock = Block.fromBlockData({}, { common })
+    const res = await vm.runBlock({
+      block: emptyBlock,
+      generate: true,
+    })
+    assert.equal(res.gasUsed, 0n)
+  })
+  it('should error when an invalid requestsRoot is provided', async () => {
+    const vm = await setupVM({ common })
+    const emptyBlock = Block.fromBlockData(
+      { header: { requestsRoot: randomBytes(32) } },
```

Comment:
Nit: does this randomness improve anything here or should we rather go with a static value here (same below)?

Reply:
Will replace with a constant value.